docs/Reference.mdx

```diff
@@ -293,11 +293,12 @@ Timestamps are expressed in seconds and display the given timestamp in the user'
 
 Guild navigation types link to the corresponding resource in the current server.
 
-| Type      | Description                                                                                           |
-|-----------|-------------------------------------------------------------------------------------------------------|
-| customize | _Customize_ tab with the server's [onboarding prompts](#DOCS_RESOURCES_GUILD/guild-onboarding-object) |
-| browse    | _Browse Channels_ tab                                                                                 |
-| guide     | [Server Guide](https://support.discord.com/hc/en-us/articles/13497665141655)                          |
+| Type         | Description                                                                                                       |
+|--------------|-------------------------------------------------------------------------------------------------------------------|
+| customize    | _Customize_ tab with the server's [onboarding prompts](#DOCS_RESOURCES_GUILD/guild-onboarding-object)             |
+| browse       | _Browse Channels_ tab                                                                                             |
+| guide        | [Server Guide](https://support.discord.com/hc/en-us/articles/13497665141655)                                      |
+| linked-roles | [Linked Roles](https://support.discord.com/hc/en-us/articles/10388356626711-Connections-Linked-Roles-Admins) tab  |
```

Comment:
```suggestion
| linked-roles | [Linked Roles](https://support.discord.com/hc/en-us/articles/10388356626711) tab  |
```

Also, advaith mentioned it also supports <id:linked-roles:LINKED_ROLE_ID>: https://x.com/advaithj1/status/1827561928606281866